lonboard/experimental/_layer.py

```diff
@@ -329,3 +353,333 @@ class TextLayer(BaseArrowLayer):
     default [0, 0]
     """
     #  ?: Accessor<DataT, [number, number]>;
+
+
+class TripsLayer(BaseArrowLayer):
+    """The `TripsLayer` renders animated paths that represent vehicle trips.
```

Comment:
not just vehicles!

Reply:
Generally I just copy that docstring from deck.gl (https://deck.gl/docs/api-reference/geo-layers/trips-layer), but in this case maybe we should choose a different description